### DISCUSSION TITLE
Hey Guys. Virus

### DISCUSSION THREAD

**Discussion by Grastice** [community/community#32828] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey i am sorry for this. I don't know where to talk about this but i think there is someone in this community bad and evil.

He tries to install some cheap malware or some kind of virus in to your computer but thanks to antiviruses that installment prevented.
The download adress points here and so i am too.I will drop the link here guys please DO NOT CLICK ON THIS AND IF ITS POSSIBLE REMOVE IT OR TELL THE ADMINS TO DO SO.

OKAY HERE IS THE BAD LINK
https://codeload.github.com/plainsoftupdt/Download-Setup-58/zip/refs/heads/main

**Comment by augustoaraujoo** [comment]
thank you man

**Comment by Nukesor** [comment, marked as answer]
Take a look at  https://github.com/community/community/discussions/32827#discussioncomment-3630395

The answer over there explains how to file a proper malware report :)
